### DISCUSSION TITLE
[Feature] Schedule photo upload to immich from App

### DISCUSSION THREAD

**Discussion by juanpeich** [immich-app/immich#11674] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In order to avoid having to regret uploading photos taken incorrectly, or photos taken only for momentary use, it would be interesting to be able to program when we choose to upload the photos.

**Utility:**
Currently, you can configure the delay from when the photo is taken as well as if we want it to be uploaded only via WiFi.

I propose that you can configure when we want it to be uploaded, being able to choose:

- Day of the week, or every day.

- The time that they will be uploaded

For example, setting an automatic upload every day at 3:00 AM, this is interesting so that the copies are made while we sleep, which has already given time for the invalid photos to be reviewed, and saves us from having to enter the server later to delete them.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#9008
